### PR TITLE
Add bundle measure (fractal dimension)

### PR DESCRIPTION
**Pull request by yuzheww**

### CODE REVIEW

File: scilpy/tractanalysis/reproducibility_measures.py [hunk 1]
```diff
@@ -179,6 +179,57 @@ def approximate_surface_node(roi):
...
+                              box_size_max=None, base=2.0,
+                              threshold=0.0, box_size=None):
+    """
+    Compute the fractal dimension of a bundle to measure the roughness.
+    The code is extracted from https://github.com/FBK-NILab/fractal_dimension
+    Parameters
+    ----------
+    density: ndarray
+        A ndarray where voxel values represent the density of a bundle.
+    n_steps: int
```
Comment (frheault) [inline scilpy/tractanalysis/reproducibility_measures.py:192]: No description

File: scilpy/tractanalysis/reproducibility_measures.py [hunk 2]
```diff
@@ -179,6 +179,57 @@ def approximate_surface_node(roi):
...
+    box_size_min: float
+        The minimum size of a box.
```
Comment (frheault) [inline scilpy/tractanalysis/reproducibility_measures.py:194]: This description does not help to understand the parameters.
What is its influence, what is a good range, how to choose it, when/why would it be better to fix it manually?
This apply to all parameters below

**Review by frheault — APPROVED** [review]
LGTM!

Last thing to add, could you add a similar warning to the docstring of the script itself too (scil_evaluate_bundles_individual_measures.py). Since the end user will likely read only this part.

**Comment by arnaudbore** [comment]
@yuzheww please add this new option in the test of the script.

**Comment by yuzheww** [comment]
> @yuzheww please add this new option in the test of the script.

Do you mean I should add a new argument for this measure in this script?

**Pull request merged**